Week-4/Javascript_basics/maps/script.js

```diff
@@ -0,0 +1,21 @@
+const calcWordFrequencies = () => {
+  let question = prompt('Enter the input');
+  if (question) {
```

Comment:
```suggestion
  if (question?.length) {
```

Reply:
Is .length needed, because i am checking if there is input for the prompt or not?

Reply:
this was to handle '' case

Reply:
Hmm